src/GafferModule/GafferModule.cpp

```diff
@@ -187,6 +193,25 @@ void nameProcess()
 #endif
 }
 
+#ifdef _MSC_VER
+void verifyAllocator()
+{
+
+	char **replacementLog;
+	int replacementStatus = TBB_malloc_replacement_log( &replacementLog );
+
+	if( replacementStatus != 0 )
+	{
+		IECore::msg( IECore::Msg::Debug, "Gaffer", "Failed to install TBB memory allocator. Performance may be degraded." );
+		for( char **logEntry = replacementLog; *logEntry != 0; logEntry++ )
+		{
+			IECore::msg( IECore::Msg::Info, "Gaffer", *logEntry );
+		}
```

Comment:
Seems a bit odd that the primary message ("Failed to install") is given a lower log level than the detailed messages. Perhaps `WARNING` for the first, and `DEBUG` for the others, or even warning for all? Now I think about it, it does seem worth having whatever we print be visible at the default logging level, since we definitely don't expect or want this to fail.